index.js

```diff
@@ -77,6 +77,10 @@ function sanitizeHtml(html, options, _recursing) {
       allowedAttributesMap[tag].push('class');
     }
 
+    classes.map(function(clss) {
+      return quoteRegexp(clss).replace(/\\\*/g, '.*')
```

Comment:
This doesn't look right. The return value of the map call is not used. Therefore classes is unmodified, unless I'm missing something. Has this been tested?